code/controllers/subsystem/runechat.dm

```diff
@@ -0,0 +1,14 @@
+TIMER_SUBSYSTEM_DEF(runechat)
```

Comment:
lohikar will remember this

Reply:
runechat uses its own timer for [qdel callback](https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/6867/files#diff-482c8a4a3a2f4e917e93ddf6eb74e7b58bbfe3a1ed3eac6bb606dd2fa170909eR255)